classes/controller/FrontController.php

```diff
@@ -1549,7 +1549,10 @@ public function getTemplateVarConfiguration()
 
     protected function getDisplayTaxesLabel()
     {
-        return (Module::isEnabled('ps_legalcompliance') && (bool) Configuration::get('AEUC_LABEL_TAX_INC_EXC')) || $this->context->country->display_tax_label;
+        return
+            (Module::isEnabled('ps_legalcompliance') && (bool) Configuration::get('AEUC_LABEL_TAX_INC_EXC'))
+            || (Configuration::get('PS_TAX_DISPLAY') && $this->context->country->display_tax_label)
```

Comment:
⚠️ I think that `PS_TAX_DISPLAY` controls whether or not _PS displays the tax amount on a dedicated line_ (in order to show the customer how much of his money is dedicated to taxes), but not whether or not PS taxes are enabled. The configuration key you are looking for is `PS_TAX`.

![Capture d’écran 2019-04-16 à 11 20 29](https://user-images.githubusercontent.com/3830050/56197834-f87f3780-6039-11e9-9e2b-2bb3b76f246a.png)


For me, first line controls `PS_TAX` and second line controls `PS_TAX_DISPLAY`

Please double-check this 😅 as these names are really ambiguous.

Reply:
I noticed those two configuration, but actually when `PS_TAX` is disabled it automatically set `PS_TAX_DISPLAY` to false. So I thought it made sense to use this configuration, besides if you want to keep using tax but not display them you would use this configuration as well.
It's a bit confusing indeed, @colinegin what do you think is the right configuration to use?

Reply:
If you have `PS_TAX` enabled but `PS_TAX_DISPLAY` it does not mean "I use taxes but I do not display them", it means "I use taxes but I dont want to display a line for taxes, only write "included" or "excluded" on the cart total".

`PS_TAX_DISPLAY` is a very bad name, actually it should be `PS_TAX_DISPLAY_A_TAX_AMOUNT_LINE_FOR_CHECKOUT`. It only controls the display of a line, but there's a whole tax-dedicated behavior that just does not care about it.

Reply:
You're right! I changed it and use `PS_TAX`